src/components/modal/DeleteResourceModal.tsx

```diff
@@ -73,11 +73,9 @@ export const DeleteResourceModal: React.FC<DeleteResourceModalProps> = ({
       }) => {
         const input = values.resourceName;
         const isValid = input === resourceName;
-        const helpText = (
-          <FormHelperText className="pf-m-warning" isHidden={!(touched && !input)}>
-            {obj.kind} name missing
-          </FormHelperText>
-        );
+        const helpText = !(touched && !input) ? (
```

Comment:
Should be shown only when field is empty I think
```suggestion
        const helpText = (touched && !input) ? (
```